cmd/main.go

```diff
@@ -45,6 +48,8 @@ func main() {
 		log.Fatal(err)
 	}
 
+	server.AddHandler("HCS_TOPIC_MSG", cmh.NewConsensusMessageHandler(*messageRepository))
```

Comment:
Let's keep the code consistent:
1. Create db, clients, repositories
2. Create server
3. Add CryptoTransfer handler + watcher
4. Add HCS handler + watcher